app/scripts/modules/titus/src/migration/titusMigrationConfigurer.component.html

```diff
@@ -23,7 +33,7 @@
         </label>
         <input class="form-control input-sm" type="number" min="0"
                ng-model="$ctrl.config.config.waitTime"
-               ng-change="$ctrl.configChanged()"/> seconds between updates
+               ng-change="$ctrl.configChanged()"/> <span style="font-size: 12px">seconds between updates</span>
```

Comment:
![image](https://user-images.githubusercontent.com/388652/37811787-4f6e36da-2e19-11e8-81ee-c9f8fb39a7f5.png)

Quick hack to fix this wrapping?